tests/unittests/chronikService.test.ts

```
@@ -1269,3 +1270,251 @@ describe('Additional behavior and integration tests', () => {
+    // second call => succeed
+    txMock.mockImplementationOnce(async () => ({ txid: 'txABC', inputs: [], outputs: [] }))
+
+    const tx = await (client as any).fetchTxWithRetry('txABC', 3, 1)
+
+    expect(tx).toBeDefined()
+    expect(tx.txid).toBe('txABC')
+    expect(txMock).toHaveBeenCalledTimes(2)
+  })
+
+
+  it('clearOldMessages expires entries by TTL and from the correct maps', () => {
```

ESLint

[error] More than 1 blank line not allowed.

(no-multiple-empty-lines)